src/plugins/core/plugin_minify_html_literals.ts

```diff
@@ -0,0 +1,33 @@
+import { minifyHTMLLiterals } from 'minify-html-literals';
+import { Context } from '../../core/Context';
+
+export function pluginMinifyHtmlLiterals(...options: any) {
+  const useroptions = options;
+  return (ctx: Context) => {
+    ctx.ict.on('assemble_fast_analysis', props => {
+      const { module } = props;
+      if (module.props && (module.props.extension === '.ts' || module.props.extension === '.js')) {
+        if (module.contents.indexOf('html`') !== -1 || module.contents.indexOf('css`') !== -1) {
```

Comment:
we can remove that probably

Reply:
I figured it was best to check first. If there is not template tags with those names we dont need to run the plugin, indexof will be faster then using the plugin. 
Want to keep fusebox fast :-)